code/datums/diseases/transformation.dm

```diff
@@ -123,14 +122,11 @@
 					"<span class='warning'>You have a craving for bananas.</span>", "<span class='warning'>Your mind feels clouded.</span>")
 	stage5 = list("<span class='warning'>You feel like monkeying around.</span>")
 
-/datum/disease/transformation/jungle_fever/do_disease_transformation(mob/living/carbon/affected_mob)
-	if(affected_mob.mind && !IS_INFECTED_MONKEY(affected_mob.mind))
-		affected_mob.mind.add_antag_datum(/datum/antagonist/monkey)
+/datum/disease/transformation/jungle_flu/do_disease_transformation(mob/living/carbon/affected_mob)
+	if(affected_mob.mind)
```

Comment:
does this still need that mind check then? it just turns you into a monkey, doesnt give you antag anymore

Reply:
![so-true](https://user-images.githubusercontent.com/40974010/147861534-1b1eb3b0-cf4f-422b-aff5-0175200eaaf6.gif)